data/helpers.d/backend

```diff
@@ -51,3 +51,41 @@ ynh_remove_logrotate () {
 		sudo rm "/etc/logrotate.d/$app"
 	fi
 }
+
+# Create a dedicated nginx config
+#
+# usage: ynh_add_nginx_config
+ynh_add_nginx_config () {
+	finalnginxconf="/etc/nginx/conf.d/$domain.d/$app.conf"
+	ynh_compare_checksum_config "$finalnginxconf" 1
```

Comment:
No simply to prevent to trashed a modified config by the user.
In fact, this line is designed to be used in the upgrade script.